example/src/screens/CustomHtmlUI.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
+import { View, Platform, StyleSheet } from 'react-native';
+import { useFocusEffect } from '@react-navigation/native';
+import {
+  Event,
+  usePlayer,
+  PlayerView,
+  SourceType,
+} from 'bitmovin-player-react-native';
+import { useTVGestures } from '../hooks';
+
+function prettyPrint(header: string, obj: any) {
+  console.log(header, JSON.stringify(obj, null, 2));
+}
+
+export default function BasicPlayback() {
+  useTVGestures();
+
+  const player = usePlayer({
+    styleConfig: {
+      playerUiCss:
+        'https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.css',
+      playerUiJs: 'https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.js',
```

Comment:
As mentioned above, you can use this URL for the example screen:
```suggestion
      playerUiCss:
        'https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.css',
      playerUiJs: 'https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.js',
      supplementalPlayerUiCss:
        'https://storage.googleapis.com/bitmovin-player-cdn-origin/player/ui/ui-customized-sample.css'
```

Reply:
@joornby could you make this edit?

Reply:
@rolandkakonyi Yes!  I was out this week, but I'll get to this first thing Monday morning.